plugin/beacon.vim

```diff
@@ -225,7 +225,7 @@ endfunction
 let s:prev_cursor = 0
 " highlight position if cursor moved significally
 function! s:Cursor_moved()
-    let l:cur = line(".")
+    let l:cur = winline()
```

Comment:
Cool! That what I was looking for :) You can actually remove code below, that checks for folds, because it will now ignore folds without additional checks. Now if statement on line 231 can look like this
```
if l:diff > g:beacon_minimal_jump
    call s:Highlight_position(v:false)
endif
```

Reply:
I see!